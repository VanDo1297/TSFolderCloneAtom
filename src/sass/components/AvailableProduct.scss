.available-product{
  width: 300px;
  margin: 10px;
  text-align: center;
  display: flex;
  justify-content: center;
  border-radius: 10px;
  background-color: #fff;
  box-shadow: 0 9px 32px 0 rgba(0,0,0,.2);
  .content{
    padding: 30px;
    .title,.price{
      font-size: x-large;
    }
    .title,.price{
      padding-bottom: 15px;
    }
    .purchase-button{

    }
  }
}