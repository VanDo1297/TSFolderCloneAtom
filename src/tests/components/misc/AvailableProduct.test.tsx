import React from 'react';
import renderer from 'react-test-renderer';
import AvailableProduct from "../../../components/misc/AvailableProduct";

test('Renders Correctly', () => {
    const component = renderer.create(
        <AvailableProduct title={"Test"} id={"SomeID"} price={123} purchaseText={"Purchase"} onPurchase={()=>false}/>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
    console.log(tree);
});