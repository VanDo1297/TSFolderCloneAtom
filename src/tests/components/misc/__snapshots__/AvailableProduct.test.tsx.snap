// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Correctly 1`] = `
<div
  className="available-product"
>
  <div
    className="content"
  >
    <div
      className="title"
    >
      Test
    </div>
    <div
      className="price"
    >
      $
      123
    </div>
    <button
      className="purchase-button btn btn-primary"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      Purchase
    </button>
  </div>
</div>
`;
